source/quick-start.txt

```diff
@@ -0,0 +1,136 @@
+.. _entity-framework-quickstart:
+
+===========
+Quick Start
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/quick-start/overview.rst
+
+Create a MongoDB Cluster
+------------------------
+
+.. procedure::
+   :style: connected
+
+   .. step:: Set Up a Free Tier Cluster in Atlas
+
+      To set up your Atlas free cluster required for this Quick Start, complete the guidei
+      on :guides:`MongoDB Atlas Setup </atlas/account>`.
+
+      After completing the steps in the Atlas guide, you have a new MongoDB
+      cluster deployed in Atlas, a new database user, and
+      sample datasets loaded into your cluster. You also have
+      a connection string similar to the following in your copy buffer:
+
+      .. code-block:: bash
+         :copyable: false
+
+         "mongodb+srv://<username>:<password>@cluster0.abc.mongodb.net/?retryWrites=true&w=majority"
+
+   .. step:: Update the Placeholders
+
+     Paste the connection string in your copy buffer into a file in your preferred text
+     editor. Replace the ``<username>`` and ``<password>`` placeholders with
+     your database user's username and password.
+
+     Save this file to a safe location for use in the next step.
+
+   .. step:: Set Your Connection String
```

Comment:
Suggestion:
I think the connection between copying your connection string to an environment variable and "set your connection string" seems more distant than necessary. I think "Add your Connection String to an Environment Variable" would more accurately describe this step.